features/evidence.feature

```diff
@@ -75,12 +75,15 @@ Scenario: Return to dashboard
   And I am on an application waiting for evidence
   When I have successfully submitted the evidence
   And I complete processing
+  And I click on back to start
   Then I should be taken back to my dashboard
   And the application should have the status of processed
 
-@wip
-Scenario: You cannot edit any details
-  And I have successfully processed the evidence
+  @wip
```

Comment:
Is this scenario still work in progress?

Reply:
Yes - I have made a ticket for the fix: https://tools.hmcts.net/jira/browse/RST-2913